### ISSUE TITLE
Skipped Tests, failed BeforeMethod or other befores, reason for skipping, video recording, logs

### ISSUE THREAD

**Issue by chrberSTO** [telekom/testerra#84]
**Is your feature request related to a problem? Please describe.**
Whenever a test has a e.g. BeforeMethod, the test gets skipped if something goes wrong during its execution.

**Describe the solution you'd like**
It would be nice to have some reason why the test was skipped, ideally there would be a video just as in a test and log output, so that one can pinpoint the reason for a skip.

**Describe alternatives you've considered**
I shipped around this by including the code that was previously executed in the before into the actual test. But that defeats the purpose of the before annotations.

**Additional context**
I have sporadic failures of tests because something went wrong in the before. This could be because there is a problem with other tests run in parallel or something else, I just do not know. I guess one could analyze the logs, maybe there is something there.

**Comment by mreiche** [comment]
In `report-ng` you have the dependency view, which should give you the hint you are missing.

**Comment by chrberSTO** [comment]
Hi Mike,

 

Yes, this is exactly what I need. Will both reports be there side by side forever, will they me married at some point or will just one stay there? If so, which one?

 

Thank you for your help.

Kind regards.

Christian

 

From: Mike Reiche ***@***.***>
Reply to: telekom/testerra ***@***.***>
Date: Thursday, 1. July 2021 at 11:02
To: telekom/testerra ***@***.***>
Cc: chrberSTO ***@***.***>, Author ***@***.***>
Subject: Re: [telekom/testerra] Skipped Tests, failed BeforeMethod or other befores, reason for skipping, video recording, logs (#84)

 

In report-ng you have the dependency view, which should give you the hint you are missing.

**Comment by mreiche** [comment]
No, the are not a couple. more like far relatives and the older generation will be left for dying soon.
Please use `report-ng` and tell us what you are missing compared to `report` and please come back, if the Dependency View fits your needs.

**Comment by chrberSTO** [comment]
Hi Mike,

 

This is exactly what I was looking for, well done, it works great.

 

Thank you, this feature request can be closed.

 

Thanks.

Christian

 

 

From: Mike Reiche ***@***.***>
Reply to: telekom/testerra ***@***.***>
Date: Thursday, 1. July 2021 at 13:34
To: telekom/testerra ***@***.***>
Cc: chrberSTO ***@***.***>, Author ***@***.***>
Subject: Re: [telekom/testerra] Skipped Tests, failed BeforeMethod or other befores, reason for skipping, video recording, logs (#84)

 

No, the are not a couple. more like far relatives and the older generation will be left for dying soon.
Please use report-ng and tell us what you are missing compared to report and please come back, if the Dependency View fits your needs.
